src/routes/user/[user].svelte

```diff
@@ -1,20 +1,32 @@
 <script context="module">
 	export async function load({ params }) {
-		let user = params.user;
-		return { props: { user } };
+		let usr = params.user;
+		return { props: { usr } };
 	}
 </script>
 
-<script>
+<script lang="ts">
 	import Post from '../../components/Post.svelte';
 
-	export let user;
-	export let avatar;
-	export let banner;
-	export let name;
-	export let username = user;
-	export let aboutMe;
-	export let isFollowing = false;
+	let usr: string;
```

Comment:
Change usr to user for clarity

Reply:
will change the name to `path` since it's the url string and doesn't have much of the user